front-end/src/components/newsfeed-view/news-list/news-card/newscard.scss

```diff
@@ -0,0 +1,110 @@
+
+.news_card {
+
+    margin: 15px 0px;
```

Comment:
spacing, typo, and color should rely on a system. arbitrary definition will come back at you in the long run

Reply:
yes, I totally agree. That's something I need to redo